models/aave/ethereum/aave_v2_borrow.sql

```diff
@@ -0,0 +1,137 @@
+{{ config(
+  materialized='view'
+  , alias='borrow'
+  , post_hook='{{ expose_spells(\'["ethereum"]\',
+                                  "project",
+                                  "aave_v2",
+                                  \'["batwayne", "chuxinh"]\') }}'
+  )
+}}
+
+SELECT
+      version,
+      transaction_type,
+      loan_type,
+      erc20.symbol,
+      borrow.token as token_address,
+      borrower,
+      repayer,
+      liquidator,
+      amount / concat('1e',erc20.decimals) AS amount,
+      (amount/ concat('1e',p.decimals)) * price AS usd_amount,
+      evt_tx_hash,
+      evt_index,
+      evt_block_time,
+      evt_block_number   
+FROM (
+SELECT 
+    '1' AS version,
+    'borrow' AS transaction_type,
+    CASE 
+        WHEN _borrowRateMode = '1' THEN 'stable'
+        WHEN _borrowRateMode = '2' THEN 'variable'
+    END AS loan_type,
+    CASE
+        WHEN _reserve = '\xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE' THEN '\xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' 
```

Comment:
Format addresses according to v2 conventions (0x... and lowercase)

Reply:
Done